### ISSUE TITLE
Issue with Vector Drawables on Jenkins

### ISSUE THREAD

**Issue by cazeit** [takahirom/roborazzi#167]
I am currently integrating Roborazzi into a project. So far everything was straight forward and I love the approach!
However, I am currently facing an issue I cannot explain, which I guess should be caused by Roboelectric's behaviour rather than Roborazzi. I would love to hear if someone struggled with this before.

On my local machine (ARM Mac), the screenshot recording and verification work completely fine. But on Jenkins (x86_64 Mac) the recorded screenshots all have wrongly rendered vector drawables (consistently). I added some screenshots to display the issue. JDK, Build tools and Gradle have the same version on both machines. What else could cause this?

Thank you so much! I apologize if this is seen as unrelated to this repository.

| Local (correct)  | Jenkins (incorrect) |
| ------------- | ------------- |
| ![example_drawable_1_correct](https://github.com/takahirom/roborazzi/assets/49943844/a9f04ed5-de13-4269-b7a8-23bb1f166371)  | ![example_drawable_1_incorrect](https://github.com/takahirom/roborazzi/assets/49943844/7702b83b-bfb3-4e00-b22b-5e42ab0c9b4f)  |
| ![example_drawable_2_correct](https://github.com/takahirom/roborazzi/assets/49943844/34d67cce-0caa-4bf7-862e-073a8089102c)  | ![example_drawable_2_incorrect](https://github.com/takahirom/roborazzi/assets/49943844/64190899-1701-495a-a66a-ab2a5c6be3a2)  |

**Comment by takahirom** [comment]
Thank you for the report! That's an interesting bug. 👀
Could you please verify if you have set the SDK version using @Config(sdk = [33])?

This is because ShadowVectorDrawable is utilized for SDK versions 24 and above

```kotlin
@Implements(value = VectorDrawable.class, minSdk = N)
public class ShadowVectorDrawable
```

**Comment by cazeit** [comment]
Yeah, truly interesting! 🤔 

Yes, my tests are typically annotated like this:
```kotlin
@RunWith(RobolectricTestRunner::class)
@GraphicsMode(GraphicsMode.Mode.NATIVE)
@HiltAndroidTest
@Config(
  application = HiltTestApplication::class,
  sdk = [33],
  qualifiers = RobolectricDeviceQualifiers.Pixel6Pro,
)
```

**Comment by takahirom** [comment]
In tracing the source code, it goes from RNG to VectorDrawable
https://mvnrepository.com/artifact/org.robolectric/nativeruntime-dist-compat/1.0.1
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/core/jni/Android.bp#390
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/core/jni/Android.bp#435
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/libs/hwui/Android.bp#627
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/libs/hwui/Android.bp#632
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/libs/hwui/Android.bp#338

It appears to be using Skia, although I'm not sure why that happens
https://android.googlesource.com/platform/frameworks/base/+/refs/heads/android12-hostruntime-dev/libs/hwui/VectorDrawable.cpp#24

It can related to this.
https://github.com/robolectric/robolectric/issues/8457

**Comment by takahirom** [comment]
Since there's nothing that can be done on the Roborazzi end, and the issue might be due to differences in CPU architecture or data lengths of integers, I don't think I can resolve it. Therefore, I'll go ahead and close this issue.

**Comment by cazeit** [comment]
For anyone who stumbles accross this issue:
This behaviour is fixed with Roboelectric release 4.14.
